plugin-api/src/main/java/net/clgd/ccemux/api/config/Config.java

```diff
@@ -1,19 +1,23 @@
 package net.clgd.ccemux.api.config;
 
-import com.google.common.reflect.TypeToken;
+import javax.annotation.Nonnull;
 
-import lombok.Getter;
+import com.google.common.reflect.TypeToken;
 
 /**
  * The base class for all configs, containing a root {@link Group} of entries.
  */
 public class Config {
+	private final Group root = new Group();
+
 	/**
 	 * The root group for the config. All properties and child groups are stored in
 	 * this.
 	 */
-	@Getter
-	private final Group root = new Group();
+	@Nonnull
+	public Group getRoot() {
```

Comment:
There's a bit of me which would just call this `root()`, though I realise it doesn't fit java beans or w/e.